src/core/network/qgshttpheaders.cpp

```diff
@@ -0,0 +1,111 @@
+/***************************************************************************
+                       qgshttpheaders.cpp
+  This class implements simple http header management.
+
+                              -------------------
+          begin                : 2021-09-09
+          copyright            : (C) 2021 B. De Mezzo
+          email                : benoit dot de dot mezzo at oslandia dot com
+
+***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgshttpheaders.h"
+
+//
+// QgsHttpHeaders
+//
+
+const QString QgsHttpHeaders::KEY_PREFIX = "http-header/";
+
+QgsHttpHeaders::QgsHttpHeaders()
+{
+  // nope
```

Comment:
?

Reply:
comment when the block is intentionally empty: no operation